src/utils/[questionId]ApiClient.ts

```diff
@@ -0,0 +1,100 @@
+import axios from "axios";
```

Comment:
please move the file to a `src/lib/api` folder and rename to `questionClient.tsx`